.gitignore

```diff
@@ -85,4 +85,5 @@ lint/tmp/
 # lint/reports/
 
 # Misc
-*.DS_Store
\ No newline at end of file
+*.DS_Store
+.idea/
```

Comment:
Given the convention, we should not ignore .idea

Reply:
Ok, I put it because I just had an update that changed a couple files, but I can change that back